js/app.js

```diff
@@ -44,55 +19,21 @@ window.addEventListener('DOMContentLoaded', function() {
   var messageArea = document.getElementById('message-container');
 
   function enableImport() {
-    var timeEnable;
-    if (!accessToken || !tokenValidity) {
-      timeEnable = -1
-    } else {
-      timeEnable = tokenValidity - Date.now();
-    }
-
-    if (timeEnable > 0) {
-      importButton.style.display = '';
-      authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
-        'reauthorize';
+    importButton.style.display = '';
+    authorizeButton.querySelector('div[data-l10n-id]').dataset.l10nId =
+      'reauthorize';
+    navigator.mozL10n.ready(() => {
       navigator.mozL10n.translate(authorizeButton);
-      setTimeout(disableImport, timeEnable);
-    } else {
-      disableImport();
-    }
+    });
```

Comment:
Maybe I can try to explain what was my intention: I want to change the wording of this button when we have a valid authorization token vs when we have not. I first tried to change only the data-l10n-id but I figured that mozL10n did not pick my change up at runtime (I expected that). So I thought that I needed to tell mozL10n that this element needs to be translated again.

My knowledge of l10n.js is limited, so there may very well be a better way. Alternatively, I can have 2 elements, one with the "Authorize", one with the "Reauthorize" and hide/show them when needed. WDYT?